### PR TITLE
Always cleanup the rails db

### PR DESCRIPTION
**Pull request by rsalevsky**
Requires https://github.com/crowbar/crowbar-core/pull/198

Fixes https://github.com/crowbar/crowbar/pull/2203

### CODE REVIEW

**Comment by MaximilianMeister** [comment]
could you start a gating with the dependent https://github.com/crowbar/crowbar-core/pull/198 ?

File: scripts/install-chef-suse.sh [hunk 1]
```diff
@@ -278,6 +278,15 @@ post_fail_handler ()
     rm -f $crowbar_install_dir/crowbar_installing
 }
 
+function reset_crowbar()
+{
+    rm -f $crowbar_install_dir/crowbar-install-failed
+    rm -f $installation_steps
+    pushd /opt/dell/crowbar_framework > /dev/null
+    bin/rake db:cleanup
```
Comment (vuntz) [inline scripts/install-chef-suse.sh]: Is it safe to run as root, or should it be run as the crowbar user?

Comment (tboerger) [inline scripts/install-chef-suse.sh]: That should not matter as it's not creating files, it's just migrating down and up an existing database.

**Comment by rsalevsky** [comment]
@MaximilianMeister @vuntz @tboerger Gating is green, can you please review?

**Comment by MaximilianMeister** [comment]
+1

**Comment by vuntz** [comment]
+1

**Pull request merged**